lms/views/basic_lti_launch.py

```diff
@@ -37,13 +37,6 @@ def __init__(self, context, request):
         self.context = context
         self.request = request
 
-        self.context.js_config.config.update(
-            {
-                # Configure the front-end mini-app to run.
-                "mode": "basic-lti-launch",
-            }
-        )
```

Comment:
"basic-lti-launch" is now the default, so no need to set this here

Reply:
It's technically unnecessary, but I wonder if it would be nice to have to set it here as well (not like this of course). This would mean:

 * Everything behaved the same regardless of mode
 * You'd know the mode existed and what value it would be set to just by looking at the view
    *  Rather than having to know it existed and that there was a default set in the JSConfig

This would be easily accommodated with a parameterised `mode` or `set_mode` etc.

Reply:
It would be a little weird though because `BasicLTILaunchViews.__init__()` would be explicitly setting the mode to `"basic-lti-launch"` and then `BasicLTILaunchViews.unconfigured_basic_lti_launch()` would be changing it to `"content-item-selection"`.

I think it'd just be more code. `BasicLTILaunchViews.__init__()` would need to set the mode and need a test that it sets the mode. And `JSConfig` would need a way of setting the mode to  `"basic-lti-launch"`, whether a separate method or by complicating the `enable_content_item_selection_mode()` method (which in future is going to grow a bunch of content item selection-specific params), and tests for that.